include/icp/driver.h

```diff
@@ -21,6 +21,7 @@ namespace icp {
 
             /** The transform. */
             RBTransform transform;
+            ConvergenceState(int dim): transform(dim) {}
```

Comment:
`size_t` would be appropriate here, I think.
```suggestion
            ConvergenceState(size_t dim): transform(dim) {}
```

Reply:
Also, I don't know why it wasn't this way, but I think we should initialize `cost` and `iteration_count` to 0. I'm not sure if this current behavior is defined or not?

Reply:
changed:)